lib/pages/chat_adaptive_scaffold/chat_adaptive_scaffold.dart

```diff
@@ -0,0 +1,135 @@
+import 'dart:async';
+
+import 'package:fluffychat/pages/chat/chat.dart';
+import 'package:fluffychat/pages/chat/events/message/message_style.dart';
+import 'package:fluffychat/pages/chat_search/chat_search.dart';
+import 'package:fluffychat/utils/extension/value_notifier_extension.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
+import 'package:fluffychat/widgets/layouts/adaptive_layout/app_adaptive_scaffold.dart';
+import 'package:fluffychat/widgets/mxc_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_adaptive_scaffold/flutter_adaptive_scaffold.dart';
+import 'package:matrix/matrix.dart';
+
+class ChatAdaptiveScaffold extends StatefulWidget {
+  final String roomId;
+  final MatrixFile? shareFile;
+  final String? roomName;
+
+  const ChatAdaptiveScaffold({
+    Key? key,
+    required this.roomId,
+    this.shareFile,
+    this.roomName,
+  }) : super(key: key);
+
+  @override
+  State<ChatAdaptiveScaffold> createState() => ChatAdaptiveScaffoldController();
+}
+
+class ChatAdaptiveScaffoldController extends State<ChatAdaptiveScaffold> {
+  final showRightPanelNotifier = ValueNotifier(false);
+  final jumpToEventIdStream = StreamController<EventId>.broadcast();
```

Comment:
Don't forgot dispose `ValueNotifier` and cancel  `StreamController`

Reply:
done